src/django/api/facility_index_backfill/backfill_orchestrator.py

```
@@ -0,0 +1,170 @@
+                '--batch-size',
+                str(batch_size),
+                '--result-file',
+                result_path,
+            ]
+            if dry_run:
+                cmd.append('--dry-run')
+            self.stdout.write(
+                f'  worker {format_worker_number(worker_id)}: '
+                f'{" ".join(cmd)}'
+            )
+            processes.append(subprocess.Popen(cmd))
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)